test/integration/documentTest.tsx

```diff
@@ -1,34 +1,21 @@
 import { SimpleData, SimpleDocument, Table } from "../../src/index.js"
 import React from "react"
 import { Typography } from "@mui/material"
-import { temporaryDirectory } from 'tempy'
+import { temporaryDirectoryTask } from 'tempy'
```

Comment:
What's the pros compared to temporaryDirectory?

Reply:
I realised that you have to delete `temporaryDirectory` when you are done.  `temporaryDirectoryTask` is cleaned up after the callback is executed.  That's the behaviour we wanted from the beginning, really.